lib/src/common/variables.js

```diff
@@ -1253,32 +1255,54 @@ export const componentTokens = {
-    selectedActiveBackgroundColor: globalTokens.hal_purple_d_20,
-    selectedDisabledBackgroundColor: globalTokens.hal_purple_l_95,
-    selectedFontColor: globalTokens.hal_white,
-    selectedDisabledFontColor: "#cbacec",
+    containerBackgroundColor: globalTokens.color_grey_50,
+    containerBorderColor: globalTokens.hal_grey_l_60,
+    labelFontColor: globalTokens.hal_black,
+    disabledLabelFontColor: globalTokens.hal_grey_l_60,
+    helperTextFontColor: globalTokens.hal_black,
+    disabledHelperTextFontcolor: globalTokens.hal_grey_l_60,
     unselectedBackgroundColor: globalTokens.hal_grey_l_90,
     unselectedHoverBackgroundColor: globalTokens.hal_grey_l_80,
     unselectedActiveBackgroundColor: globalTokens.hal_purple_s_38,
     unselectedDisabledBackgroundColor: globalTokens.hal_grey_l_95,
     unselectedFontColor: globalTokens.hal_black,
     unselectedDisabledFontColor: globalTokens.hal_grey_l_60,
-    iconSize: "24px",
-    iconPaddingTop: "8px",
-    iconPaddingBottom: "8px",
-    iconPaddingRight: "8px",
-    iconPaddingLeft: "8px",
-    labelPaddingTop: "8px",
-    labelPaddingBottom: "8px",
-    labelPaddingLeft: "24px",
-    labelPaddingRight: "24px",
+    selectedBackgroundColor: globalTokens.hal_purple_s_38,
+    selectedHoverBackgroundColor: globalTokens.hal_purple_d_30,
+    selectedActiveBackgroundColor: globalTokens.hal_purple_d_20,
+    selectedDisabledBackgroundColor: globalTokens.hal_purple_l_95,
+    selectedFontColor: globalTokens.hal_white,
+    selectedDisabledFontColor: globalTokens.color_purple_300,
     focusColor: globalTokens.hal_blue_l_50,
+    labelFontFamily: globalTokens.type_sans,
+    labelFontSize: globalTokens.type_scale_02,
+    labelFontStyle: globalTokens.type_normal,
+    labelFontWeight: globalTokens.type_semibold,
+    labelLineHeight: globalTokens.type_leading_loose_01,
+    helperTextFontFamily: globalTokens.type_sans,
+    helperTextFontSize: globalTokens.type_scale_01,
+    helperTextFontStyle: globalTokens.type_normal,
+    helperTextFontWeight: globalTokens.type_regular,
+    helperTextLineHeight: globalTokens.type_leading_normal,
+    optionLabelFontFamily: globalTokens.type_sans,
+    optionLabelFontSize: globalTokens.type_scale_03,
+    optionLabelFontStyle: globalTokens.type_normal,
+    optionLabelFontWeight: globalTokens.type_regular,
+    iconPaddingRight: globalTokens.spacing_03,
+    iconPaddingLeft: globalTokens.spacing_03,
+    labelPaddingLeft: globalTokens.spacing_06,
+    labelPaddingRight: globalTokens.spacing_06,
+    iconMarginRight: globalTokens.spacing_03,
+    containerMarginTop: globalTokens.spacing_02,
+    optionBorderThickness: globalTokens.border_width_0,
+    optionBorderStyle: globalTokens.border_none,
+    optionBorderRadius: globalTokens.border_radius_02,
+    containerBorderThickness: globalTokens.border_width_01,
+    containerBorderStyle: globalTokens.border_solid,
+    containerBorderRadius: globalTokens.border_radius_03,
```

Comment:
Changes on the border tokens were made #442

Reply:
Changes done!